src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1483,7 +1483,8 @@ class DocBaseViewer extends BaseViewer {
         this.pinchPage = null;
     }
 
-    toggleFindBar() {
+    toggleFindBar(elementToFocus) {
+        this.findBarToggleElement = elementToFocus;
```

Comment:
It looks like the convention in this file shortens `Element` to `El`. Should we rename to `findBarToggleEl`?

Reply:
renamed